### DISCUSSION TITLE
Single Service update on different schedule with lables.

### DISCUSSION THREAD

**Discussion by awptechnologies** [shizunge/gantry#70] [General]
So i still use the cron job to start the container. I have a single service homeassistant that seems to have a new image everynight. I am on the dev version.
Other then excluding it, is it possible to set a label so that it updates every 4th time the cron job starts the container or every 4th day?

Thats asking for specifics but this project seems to do it all. Figured i would ask.

**Comment by shizunge** [comment]
1. the simple way is to create two cron jobs.

2. you can use the pre-run command.
2.1. mount a presistant volume to share data across runs (or use sleep rather than cronjob to keep the process alive).
2.2. save a counter in a file and update teh counter in pre-run command.
2.3. based on the counter value, change the behavior.

The pre-run command runs in a subprocess in 2024.1114 release. Therefore you cannot modify Gantry configurations in the command.
In the next release, or the development version you could modify the Gantry setting in pre-run command.
